CitationStyleLanguagePlugin.inc.php

```diff
@@ -257,24 +272,43 @@ public function getCitationStyleConfig($styleId) {
 	 * @param $args array
 	 * @return false
 	 */
-	public function getArticleTemplateData($hookName, $args) {
+	public function getTemplateData($hookName, $args) {
+		$templateMgr = TemplateManager::getManager();
 		$request = $args[0];
-		$issue = $args[1];
-		$article = $args[2];
-		$publication = $args[3];
+		if ($this->isApplicationOmp()) {
+			$submission =& $args[1];
+			$publication = $submission->getCurrentPublication();
+			$issue = null;
+			if (null === $publication) {
+				return false;
+			}
```

Comment:
It shouldn't be possible for no `$publication` to exist here. The book view page would not work. Did you run into this yourself? If not, you can remove this check.